neo_api_client/neo_api.py

```diff
@@ -67,15 +68,33 @@ def __init__(self, environment="uat", access_token=None, consumer_key=None, cons
             self.configuration = neo_api_client.NeoUtility(consumer_key=consumer_key, consumer_secret=consumer_secret,
                                                            host=environment)
             self.api_client = ApiClient(self.configuration)
+
             try:
                 session_init = neo_api_client.LoginAPI(self.api_client).session_init()
                 print(json.dumps({"data": session_init}))
             except ApiException as ex:
                 error = ex
+
+        if reuse_session:
```

Comment:
checks if reuse_session is available, if yes then it sets the access_token,session_token,sid and serverId to  self.configuration